### PR TITLE
Archive Controller Accounts

### PR DESCRIPTION
**Pull request by filippoweb3**

### CODE REVIEW

**Comment by filippoweb3** [comment]
@DrW3RK @CrackTheCode016 , there is a minor thing that is missing:
- `maintain-guides-how-to-nominate-polkadot`
- `maintain-guides-how-to-nominate-kusama`
In those pages in the section about using the CLI I am not sure how to rename stuff

File: docs/maintain/maintain-guides-how-to-validate-polkadot.md [hunk 1]
```diff
@@ -618,7 +618,7 @@ people are using telemetry, it is recommended that you choose something likely t
 ### Generating the Session Keys
 
 You need to tell the chain your Session keys by signing and submitting an extrinsic. This is what
-associates your validator node with your Controller account on Polkadot.
+associates your validator node with your account on Polkadot.
```
Comment (CrackTheCode016) [inline docs/maintain/maintain-guides-how-to-validate-polkadot.md]: account, or staking proxy account?

```suggestion
associates your validator node with your staking proxy account on Polkadot.
```

File: docs/learn/learn-staking-advanced.md [hunk 2]
```diff
@@ -45,68 +45,40 @@ checkout the [Introduction to Staking](./learn-staking.md) page.
...
-If the stash and controller are different accounts, the staking proxy will just be used to bond more
-funds and change the controller. Thus the staking proxy will be used to sign for those transactions
-that are used less often and that are usually signed by the stash.
-
-![stash-controller-stakingProxy](../assets/stash-controller-stakingProxy.png)
-
-From a practical perspective, we now have two accounts and we need to remember two passwords. From a
-security perspective, the party who wants to fully control our staking actions must have access to
-two accounts.
+Note that to change the staking proxy you will need to sign with the stash or an _any_ proxy.
```
Comment (CrackTheCode016) [inline docs/learn/learn-staking-advanced.md:81]:
```suggestion
Note that to change the staking proxy you will need to sign with the stash or as _any_ proxy.
```

File: docs/learn/learn-staking-advanced.md [hunk 3]
```diff
@@ -45,68 +45,40 @@ checkout the [Introduction to Staking](./learn-staking.md) page.
...
-
-![stash-stakingProxy](../assets/stash-stakingProxy.png)
-
-This situation is similar to having the controller as a staking proxy. For a practical perspective
-we need to use only one account and remember one password to sign for all staking-related
-transactions. From a security perspective who controls the staking proxy controls our staking
-actions.
+[**extrinsic calls**](./learn-extrinsics.md/#pallets-and-extrinsics) made to specific **pallets** on
+behalf of the proxied account. There is thus the possibility to create staking proxy accounts that
+can be used to sign only extrinsic calls to staking, session and utility pallets.
```
Comment (CrackTheCode016) [inline docs/learn/learn-staking-advanced.md]:
```suggestion
can be used to sign extrinsic calls specific to the staking, session and utility pallets.
```

**Review by CrackTheCode016 — APPROVED** [review]
This overall looks good to me, thank you!

For the CLI portion, this would require a bit of experimentation to ensure it is covered.   I am leaning towards merging this PR and following the CLI portion with another PR.

File: docs/learn/learn-staking-advanced.md (hunk 2 above)
Comment (filippoweb3) [reply docs/learn/learn-staking-advanced.md:81]: "an" is right here

File: docs/maintain/maintain-guides-how-to-validate-polkadot.md (hunk 1 above)
Comment (filippoweb3) [reply docs/maintain/maintain-guides-how-to-validate-polkadot.md]: validator node is associated with the stash, I will change this

**Pull request merged**